src/main/java/frc/trigon/robot/commands/LEDAutoSetupCommand.java

```diff
@@ -58,7 +52,15 @@ public boolean runsWhenDisabled() {
 
     private Command getUpdateAutoStartPoseCommand() {
         return new InstantCommand(
-                () -> this.autoStartPose = new MirrorablePose2d(PathPlannerAuto.getStaringPoseFromAutoFile(autoName.get()), true).get()
+                () -> {
+                    try {
+                        final Pose2d nonMirroredAutoStartPose = PathPlannerPath.fromPathFile(PathPlannerAuto.currentPathName).getStartingHolonomicPose().get();
```

Comment:
why not use PathPlannerAuto.getStaringPoseFromAutoFile?
something here should be extracted

Reply:
It was removed

Reply:
Are you sure this works properly? If it was removed, are you sure this was the suggested change?